pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1367,8 +1286,8 @@ func (pl *dynamicResources) Unreserve(ctx context.Context, cs *framework.CycleSt
 		// If allocation was in-flight, then it's not anymore and we need to revert the
 		// claim object in the assume cache to what it was before.
 		if state.informationsForClaim[index].structuredParameters {
-			if _, found := pl.inFlightAllocations.LoadAndDelete(state.claims[index].UID); found {
-				pl.claimAssumeCache.Restore(claim.Namespace + "/" + claim.Name)
+			if found := pl.draManager.ResourceClaims().RemoveClaimPendingAllocation(state.claims[index].UID); found {
```

Comment:
ditto my comment above, now that we have a single bool return value, should we rename `found` to something that better describes the behavior, like `deleted`?